packages/node_modules/@ciscospark/plugin-lyra/test/integration/spec/device.js

```diff
@@ -0,0 +1,170 @@
+        lyraMachine.spark.device.config.defaults.model = `SparkBoard Test`;
+        lyraMachine.spark.device.config.defaults.localizedModel = `SparkJSSDKTest`;
+        lyraMachine.spark.device.config.defaults.systemVersion = `SparkJSSDKTest`;
+        lyraMachine.spark.device.config.defaults.systemName = `Darling`;
+
+        return lyraMachine.spark.mercury.connect();
+      })
+      .then(() => lyraMachine.spark.lyra.space.get({id: lyraMachine.id}))
+      .then((space) => {
+        lyraMachine.space = space;
+      }));
+
+    before(`create users`, () => testUsers.create({count: 2})
+      .then((users) => {
+        participants = users;
+        spock = participants[0];
+
+        return Promise.all(Array.map(participants, (participant) => {
+          participant.spark = new CiscoSpark({
+            credentials: {
+              authorization: participant.token
+            }
+          });
+
+          return participant.spark.mercury.connect();
+        }));
+      }));
+
+    before(`create conversation`, () => participants[0].spark.conversation.create({
+      displayName: `Test Lyra Conversation`,
+      participants
+    })
+      .then((c) => {
+        conversation = c;
+        return conversation;
+      }));
+
+    describe(`#getAudioState`, () => {
+      let audioState;
+
+      before(`put audio state`, () => {
+        audioState = {
+          volume: {
+            level: 2
+          },
+          microphones: {
+            muted: false
+          },
+          deviceUrl: lyraMachine.spark.device.url
+        };
+
+        return lyraMachine.spark.lyra.device.putAudioState(lyraMachine.space, audioState);
+      });
+
+      it(`returns audio state`, () => lyraMachine.spark.lyra.device.getAudioState(lyraMachine.space)
+          .then((res) => {
+            assert.equal(res.microphones.muted, audioState.microphones.muted);
+            assert.equal(res.volume.level, audioState.volume.level);
+          }));
+    });
+
+    describe(`when a call is in progress`, () => {
+      before(`ensure participant joined space`, () => spock.spark.lyra.space.join(lyraMachine.space)
+        .then(() => lyraMachine.spark.lyra.space.verifyOccupant(lyraMachine.space, spock.id))
+        .then(() => spock.spark.lyra.space.bindConversation(lyraMachine.space, conversation)));
+
+      before(`make a call`, () => {
+        const locus = {
+          url: conversation.locusUrl,
+          correlationId: uuid.v4()
+        };
+
+        return spock.spark.request({
+          method: `POST`,
+          uri: `${locus.url}/participant`,
+          body: {
+            correlationId: locus.correlationId,
+            deviceUrl: spock.spark.device.url,
+            localMedias: []
+          }
+        });
+      });
+
+      after(`remove binding`, () => spock.spark.lyra.space.unbindConversation(lyraMachine.space, conversation));
+
+      it(`allows muting`, (done) => {
```

Comment:
Please change these the imperative form "it mutes", "it unmutes", "it increases the volume", etc